lib/kadira.js

```diff
@@ -98,15 +102,15 @@ Kadira.connect = function (appId, appSecret, options) {
     Kadira.coreApi._checkAuth()
       .then(function () {
         logger('connected to app: ', appId);
-        console.log('Monti APM: Successfully connected');
+        console.log('Monti APM: Connected');
         Kadira._sendAppStats();
         Kadira._schedulePayloadSend();
       })
       .catch(function (err) {
         if (err.message === 'Unauthorized') {
-          console.log('Monti APM: authentication failed - check your appId & appSecret');
+          console.log('Monti APM: Authentication failed, check your App ID & App Secret');
```

Comment:
I'm wondering if for this message it should still refer to it as `appId` and `appSecret` since that is the name of the options.

Reply:
Yes, that makes sense, will adjust it.